nifi-frontend/src/main/frontend/apps/nifi-jolt-transform-ui/src/app/app.module.ts

```diff
@@ -54,7 +57,12 @@ if (disableAnimations !== 'true' && disableAnimations !== 'false') {
             routerState: RouterState.Minimal,
             navigationActionTiming: NavigationActionTiming.PostActivation
         }),
-        EffectsModule.forRoot(JoltTransformJsonUiEffects),
+        EffectsModule.forRoot(
+            JoltTransformJsonProcessorDetailsEffects,
+            JoltTransformJsonTransformEffects,
+            JoltTransformJsonValidateEffects,
+            JoltTransformJsonPropertyEffects
```

Comment:
These `Effects` exist within the `jolt-transform-json-ui` page which is lazy loaded. We should initialize these `Effects` within `jolt-transform-json-ui.module`.